builder/query/sql.go

```diff
@@ -15,8 +15,8 @@ type SQL struct {
 }
 
 type SQLParameter struct {
-	Type  string `json:"type,omitempty"`
-	Value string `json:"value,omitempty"`
+	Type  string      `json:"type,omitempty"`
+	Value interface{} `json:"value,omitempty"`
```

Comment:
There were cases where the type of value was INTEGER or FLOAT, so I changed it to an interface.